lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -752,9 +784,14 @@ def _rows_for_users(self, context, users, header_row, graded_scorable_blocks):
                     else:
                         earned_possible_values.append(['Not Attempted', problem_score.possible])
 
+            context.task_progress.succeeded += 1
             success_rows.append(
                 student_fields + [enrollment_status, course_grade.percent] + _flatten(earned_possible_values))
 
+        context.task_progress.update_task_state(extra_meta={'step': 'ProblemGradeReport Step 5: Calculating Grades'})
+        log_message = u'{0} {1}/{2}'.format('ProblemGradeReport Step 5: Calculating Grades',
+                                            context.task_progress.attempted, context.task_progress.total)
+        self.log_task_info(context, log_message)
```

Comment:
Will this hit for every success?

Reply:
This will be hit after successful processing of a batch i.e. **100 users processed**.